(quantchembasic)=

# Квантовая химия. Введение.

## Описание лекции

Из этой лекции мы узнаем:

- что такое квантовая химия, что с ее помощью можно сделать, а что -- нельзя;
- как устроена самая простая теория, описывающая квантовую физику атома
- какие у этой теории есть ограничения

## Введение

В предыдущих лекциях мы говорили о применении квантовой механики для вычислений и обработки информации. Однако исторически квантовая теория развивалась в первую очередь из-за того, что классическая физика не могла объяснить некоторые наблюдаемые эффекты, такие как дискретный спектр излучения атомов, фотоэффект в металлах, интерференцию частиц на дифракционных щелях.

Считается, что квантовая механика дает полное и точное описание состояния и эволюции любой системы при нерелятивистских условиях — по крайней мере, на сегодняшний день этому нет экспериментальных или теоретических противоречий. Это значит, что в теории возможно для любой системы частиц записать уравнения Шредингера, решить их и предсказать, как себя поведет система. Однако на практике оказывается, что в реальных задачах вроде моделирования лекарств и материалов просто "взять и посчитать" -- задача весьма сложная, а иногда -- неразрешимая.

Проблемами применения квантовой механики к химии и материаловедению занимается квантовая химия. Она делает это уже около 100 лет, по теме написаны толстые книжки с многоэтажными формулами, так что в лекции будут даны основы и простые примеры без полного вывода. Для желающих в конце приведены ссылки для углубленного изучения.

Предполагается, что читатель знаком с уравнением Шредингера и основными операторами (импульса, эволюции), бра-кет нотацией, а также помнит основы физики и химии на уровне старших классов школы.

## Атом водорода

### В чем проблема?

Спектр излучения и поглощения атомов — то есть на какой длине волны происходит поглощение и излучение света веществом — был одной из первых "нерешаемых" проблем, приведших в итоге к появлению квантовой физики. Для [простых веществ](https://ru.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D1%81%D1%82%D1%8B%D0%B5_%D0%B2%D0%B5%D1%89%D0%B5%D1%81%D1%82%D0%B2%D0%B0) в газообразной форме спектр является дискретным, и для атома водорода спектры поглощения и излучения в видимом диапазоне выглядят так:
```{figure} /_static/problemsblock/quantchembasic/HydrogenSpectrum.png
:name: hydrogenspectrum

Спектр поглощения и излучения водорода в видимом диапазоне
```

В конце 19 века было уже известно, что атом водорода состоит из двух заряженных частиц -- протона и электрона, но из классической физики следовало, что спектр излучения такой системы должен быть непрерывным, что никак не стыковалось с наблюдениями (дискретные линии на графике). Попробуем вслед за исследователями квантовой физики начала 20 века разобраться с этой проблемой.

```{note}
Про имена. "Квантовая физика" и "квантовая механика" -- связанные понятия, но не тождественные.

Квантовая механика -- это теория (а точнее, множество теорий с различающимися терминологиями), в основе которой лежит аксиоматика о том, что сущности описываются волновыми функциями, что они эволюционируют и, что зная эти функции, можно посчитать физические величины — например, энергию.

Квантовая физика -- это область физики, исследующая квантовые эффекты, при этом она местами может не иметь строгого обоснования -- лишь бы предсказания работали.

С квановой химией тоже есть некоторая путаница -- в двадцатом веке под ней имели в виду аналитические и численные методы решения задач квантовой механики применительно к молекулам и кристаллам. Но с развитием квантовых компьютеров эту область стали иногда называть вычислительной химией, а квантовой химией -- применение квантовых компьютеров к этой области.
```
### Быстрое и неправильное решение

Первой теорией в квантовой физике была ["Боровская теория"](https://ru.wikipedia.org/wiki/%D0%91%D0%BE%D1%80%D0%BE%D0%B2%D1%81%D0%BA%D0%B0%D1%8F_%D0%BC%D0%BE%D0%B4%D0%B5%D0%BB%D1%8C_%D0%B0%D1%82%D0%BE%D0%BC%D0%B0) -- почти классическая теория, основанная на постулатах, разрешающих противоречия классической физики с экспериментами на уровне "хардкода". Главный постулат заключается в том, что импульс может принимать только определенные дискретные значения (кванты), а не произвольные:

$ p \cdot 2\pi r = n \cdot h $

$ p = \frac{n \cdot \hbar}{ r} $

где p -- импульс, r -- радиус орбиты электрона, h -- константа Планка, n -- любое целое число.
Если применить этот постулат к атому водорода -- паре из протона и электрона, то получится, что:
- электрон вокруг протона "летает по орбите";
- спектр (уровни энергии электрона) -- дискретный.

Можно показать это следующим образом.

Во-первых, поскольку частиц две и протон имеет массу много больше массы электрона, можно перейти в почти инерциальную систему отсчета протона, где он неподвижен, и решить задачу только для электрона.

Из классической механики берем [теорему о вириале](https://ru.wikipedia.org/wiki/%D0%92%D0%B8%D1%80%D0%B8%D0%B0%D0%BB) ("для стабильной системы из двух частиц, связанных потенциальными силами, в среднем кинетическая энергия равна половине потенциальной"):

$ E_{k} = -E_{p} / 2 $

$ E = E_{k} + E_{p} = E_{p} / 2 $

Записываем кинетическую энергию и потенциал Кулона для электрона в поле протона (в СИ):

$ \frac{p^2}{2m} = k \frac{e^2}{2 R} $, где $k$ -- постоянная из закона Кулона

Используем постулат Бора и выразим "радиус" орбиты:

$ \frac{n^2 \hbar^2}{2 m R^2} = k \frac{e^2}{2R} $

$ R = \frac{n^2 \hbar^2}{k m e ^2}$

Находим полную энергию:

$ E = E_{p} / 2 = - \frac{k e^2}{ 2 R} = -\frac{k^2}{n^2} \frac{me^4}{2 \hbar^2} $

Уровни энергии пропорциональны $ \frac{1}{n^2}$ и именно [такая зависимость](https://en.wikipedia.org/wiki/Hydrogen_spectral_series) наблюдается в эксперименте.


Посчитаем минимальную энергию (n = 1):

In [1]:
from scipy import constants as consts

k = ( 1 / (4 * consts.pi * consts.epsilon_0)) # постоянная кулона

E1 = - k**2 * consts.m_e * consts.e**4 / (2 * consts.hbar ** 2) # энергия в Джоулях

E1_ev = E1 / consts.e # энергия в электрон-вольтах
print(f"Hydrogen Bohr ground state energy: {E1_ev} eV")

Hydrogen Bohr ground state energy: -13.605693122885837 eV


Получившийся ответ 13.6 eV в точности совпадает с экспериментальным значением энергии ионизации водорода.

Итого: постулировав несколько очень удачных гипотез (главная -- что импульс квантуется, то есть принимает только дискретные значения), удалось разрешить фундаментальную нестыковку между классической теорией и экспериментом: энергия стала дискретной, линии спектра стали обратно пропорциональны  $n$, значения энергии ионизации совпадают с реальными измерениями.

Казалось бы, замечательно, проблема решена! Однако теория Бора имеет ряд проблем. Главная из них заключается в том, что теория работает только для "водородоподобных" атомов, то есть состоящих из ядра и одного электрона на внешней оболочке. Уже для атома гелия спектр не согласуется с боровской теорией.

Кроме того, с точки зрения науки теория Бора -- это в некоторым смысле "читерство". Выбрав удачные постулаты и подставив их формулы в классическую физику, мы получили правильный результат. Но будь постулаты другими, результат получился бы тоже другой, так что фактически постулаты Бора -- это гениальная догадка, позволившая угадать правильные результаты для некоторых систем.